packages/vite-plugin-cloudflare/src/runner-worker/index.ts

```diff
@@ -33,6 +46,8 @@ const DURABLE_OBJECT_KEYS = [
 	'webSocketError',
 ] as const;
 
+const WORKFLOW_ENTRYPOINT_KEYS = ['run'] as const;
```

Comment:
```ts
const WORKFLOW_ENTRYPOINT_KEYS = ['run'] as const satisfies (keyof WorkflowEntrypoint)[];
```

overkill? 🤔

Reply:
I just did it like this to make it consistent with the others. I could refactor `createWorkflowEntrypointWrapper` to use the `run` method directly if you prefer but then we'll have to refactor again if they add additional methods.

Reply:
Ah yeah sorry I didn't notice we had the other `_KEYS` like this too

This is really not a big deal and I'm totally happy to keep thing consistent 👍